x/emissions/types/params.go

```diff
@@ -53,6 +53,7 @@ func DefaultParams() Params {
 		MinEffectiveTopicRevenue:            alloraMath.MustNewDecFromString("0.00000001"), // we no stop dripping from the topic's effective revenue when the topic's effective revenue is below this
 		HalfMaxProcessStakeRemovalsEndBlock: uint64(40),                                    // half of the max number of stake removals to process at the end of the block, set this too big and blocks require too much time to process, slowing down consensus
 		DataSendingFee:                      cosmosMath.NewInt(10),                         // how much workers and reputers must pay to send payload
+		MaxTopForecasterElementsToSubmit:    uint64(3),                                     // top forecast elements by score
```

Comment:
A forecaster can only retain 3 forecast elements in its forecast? 
We have a limit of 48 `MaxTopInferersToReward`. 
The way I see it, this value should be >= `MaxTopInferersToReward`

Reply:
Indeed. This is however the most expensive piece. And we'll be lowering 48-32 anyway. Asked to raise max number of elements to 12 => size of largest item in every ValueBundle is 12x12=144, which I think is a fine place to start given 32 inferers and unknown network compute bandwidth at this stage.

Reply:
should be _at most_ `MaxTopInferersToReward` but yes what @RedBird96 said